src/main/java/hudson/plugins/git/Revision.java

```diff
@@ -121,14 +121,14 @@ public String toString() {
 
     @Override
     public Revision clone() {
-        Revision clone;
         try {
-            clone = (Revision) super.clone();
+            Revision clone = (Revision) super.clone();
+            clone.branches = new ArrayList<>(branches);
+            return clone;
         } catch (CloneNotSupportedException e) {
             throw new RuntimeException("Error cloning Revision", e);
         }
-        clone.branches = new ArrayList<>(branches);
-        return clone;
+        
```

Comment:
The ci job fails with the message `[ERROR] Run 'mvn spotless:apply' to fix these violations.`.  The pull request checklist includes the item 

* [ ] I have read the [CONTRIBUTING](https://github.com/jenkinsci/git-client-plugin/blob/master/CONTRIBUTING.adoc) doc

The [contributing guide](https://github.com/jenkinsci/git-client-plugin/blob/master/CONTRIBUTING.adoc#code-formatting) says:

> Source code and pom file formatting is maintained by the spotless maven plugin. Before submitting a pull request, confirm the formatting is correct with:
> 
> mvn spotless:apply

```suggestion
```

Reply:
I'll do that.

Reply:
Done